src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -333,7 +390,8 @@ public static boolean positionForbidden(double armPos, double wristPos) {
         boolean ground = tip.getY() < -ARM_JOINT_TOTAL_HEIGHT;
 
         //return horizontal && vertical || ground;
-        return false;
+        forbFlag = ((horizontal && vertical) || ground);
+        return forbFlag;
```

Comment:
```suggestion
        return (forbFlag = ((horizontal && vertical) || ground));
```
The intermediary `forbFlag` here is confusing.

Reply:
Nevermind, ignore this comment.